tests/unit/s2n_malformed_handshake_test.c

```diff
@@ -240,6 +240,7 @@ int main(int argc, char **argv)
 
     EXPECT_NOT_NULL(conn = s2n_connection_new(S2N_CLIENT));
     EXPECT_NOT_NULL(config = s2n_config_new());
+    EXPECT_SUCCESS(s2n_config_set_cipher_preferences(config, "test_all"));
```

Comment:
I don't see any explicit cipher manipulation here. Why is "test_all" necessary?

Reply:
Removed.